app/views/posts/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with(model: @post, url: posts_path, method: :post) do |form| %>
+  <div class="field">
+    <%= form.label :title %>
+    <%= form.text_field :title %>
+  </div>
+
+  <div class="field">
+    <%= form.label :content %>
+    <%= form.text_area :content %>
+  </div>
+
+  <div class="actions">
+    <%= form.submit 'Create Post' %>
+  </div>
+<% end %>
```

Comment:
![image](https://github.com/Kidd254/BlogApp/assets/93039359/5cfd5776-4d86-4e60-975f-1d65458b9ca2)

- [ ] Great work on configuring the Post but there are some issues which are needed to be fix :thinking: Kindly fix them to make your app work as expected :+1:

1. It would be nice if you added the new method for posts in the routes.rb file because it is missing there :fearful:
2. Kindly make your post form appear even if there are no posts of a user because I see if there are no posts then I cannot access the post form :fearful: Kindly fix this issue :blush:
3. Last but not the least, when i try to create a post from your from then i see an error which you can see in the image above :eyes: Kindly fix this issue to make your app work as expected :+1: